tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -301,6 +302,22 @@ def f(b1 : bool, b2 : bool):
     assert f(True,False) == f2(True,False)
     assert f(False,True) == f2(False,True)
 
+def test_conda_flag_disable(language):
```

Comment:
Go ahead and disable this test on windows. There are warnings about compiler warnings that I can't avoid because they're not in our code.
See 
https://github.com/pyccel/pyccel/blob/f87971b2c2340f8f7310c9d447de4e85229498cd/tests/epyccel/test_external_functions.py#L10-L12
for an example